pavelib/quality.py

```diff
@@ -470,6 +516,17 @@ def _pep8_output(count, violations_list, is_html=False):
     ):
         diff_quality_percentage_pass = False
 
+    # run diff-quality for safelint.
+    if not run_diff_quality(
+            violations_type="safelint",
```

Comment:
Oh snaps. I had also thought that you just needed to give it the reports.
Looks like diff-quality is a command line tool of diff-cover, and even if you give it the reports, it needs to know how to parse them. See http://diff-cover.readthedocs.org/en/latest/#quality-coverage

A few options off the top of my head:
* Fork diff-cover, add another tool [like was done for ESLint in this PR](https://github.com/Bachmann1234/diff-cover/pull/33), use the fork and don't contribute upstream because nobody else can use safelinter as-is. :(
* Package and release the safelinter. After writing tests for it. :) This way our other repos can use it too. And other orgs. Then it would make sense to contribute another tool upstream to diff-cover.
* Fork diff-cover, [write a plug-in system for it](https://github.com/Bachmann1234/diff-cover/blob/master/CHANGELOG#L34), contribute upstream :) Then write a plug-in for the safelinter. Optionally also package and release the safelinter.